config-default.yml

```diff
@@ -235,6 +235,9 @@ guild:
         # Watch
         big_brother_logs:   &BB_LOGS        468507907357409333
 
+        # Roles
```

Comment:
There are no other roles channels, so you can maybe call this section `# Information`

Reply:
Fair enough, renamed here 72b2f803a6bb915c4bb07d3a0cd3f4d141d46431